public/static/docs/command-reference/list.md

```diff
@@ -0,0 +1,40 @@
+# list
+
+List file structure of a repo including <abbr>data artifacts</abbr>
+
+## Synopsis
+
+```usage
+usage: dvc list [-h] [-q | -v] [-R] [--outputs-only] [--rev [REV]] url [target]
+
+positional arguments:
+  url              Url to the repo. Can be either local fs path or git+ssh/https url
+  target           Path to directory within the repository to list outputs for.
+
+optional arguments:
+  --outputs-only   Show only dvc outputs.
+  --rev [REV]      Git revision (e.g. branch, tag, SHA)
+```
+
+## Description
+
+The command list files and <abbr>data artifacts</abbr> into pointed repo.
+
+## Options
+
+- `--outputs-only` - Show only <abbr>data artifacts</abbr>.
+
+- `-R`, `--recursive` - Recursively traverse the repo.
+
+- `--rev` - specific
+  [Git revision](https://git-scm.com/book/en/v2/Git-Internals-Git-References)
```

Comment:
@jorgeorpinel will need to update this when your PR is merged, or suggest the change here

Reply:
It's merged. This text seems correct except download->list and the last sentence is incomplete, should be "...by default when this option is not specified."